src/tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx

```diff
@@ -0,0 +1,47 @@
+import { fireEvent, render, screen } from '@testing-library/react'
+import { describe, expect, it, vi } from 'vitest'
```

Comment:
We should not import some things 
```suggestion
import { vi } from 'vitest'
```